src/scripts/oauth/http-refresh-token.service.js

```diff
@@ -8,22 +8,15 @@ function HttpRefreshTokenService($rootScope, $q, $injector, OAuthService, OAuthT
 
   const service = this;
   let cache = null;
-  const limiters = [
-    /www\.bungie.net\/Platform\/User\/*/,
-    /www\.bungie.net\/Platform\/Destiny\/*/
-  ];
+  const matcher = /www\.bungie.net\/Platform\/(User|Destiny)\//;
```

Comment:
Sorry for the multi-line. Just to clarify: do we want to force the URL to have a trailing slash at the end, case sensitivity?